crypto/keyring_message.go

```diff
@@ -118,6 +118,33 @@ func (keyRing *KeyRing) VerifyDetachedEncrypted(message *PlainMessage, encrypted
 	return keyRing.VerifyDetached(message, signature, verifyTime)
 }
 
+// GetVerifiedSignatureTimestamp verifies a PlainMessage with a detached PGPSignature
+// returns the creation time of the signature if it succeeds
+// and returns a SignatureVerificationError if fails.
+func (keyRing *KeyRing) GetVerifiedSignatureTimestamp(message *PlainMessage, signature *PGPSignature, verifyTime int64) (int64, error) {
+	err := keyRing.VerifyDetached(message, signature, verifyTime)
+	if err != nil {
+		return 0, err
+	}
+	return signature.getCreationTime()
+}
+
+// GetVerifiedEncryptedSignatureTimestamp verifies a PlainMessage
+// with a PGPMessage containing an encrypted detached signature
+// returns the creation time of the signature if it succeeds
+// and returns a SignatureVerificationError if fails.
+func (keyRing *KeyRing) GetVerifiedEncryptedSignatureTimestamp(message *PlainMessage, encryptedSignature *PGPMessage, decryptionKeyRing *KeyRing, verifyTime int64) (int64, error) {
```

Comment:
Do we have a use-case in mind for this? It's not needed by my side

Reply:
I don't think so, it's more for consistency but we can ditch it